bin/build.sh

```diff
@@ -15,17 +15,16 @@
 # - Z3
 # - Boogie
 # - Corral
-# - Symbooglix
 # - lockpwn
 #
 ################################################################################
 
 # Set these flags to control various installation options
 INSTALL_DEPENDENCIES=1
+INSTALL_RUST=1
 BUILD_Z3=1
 BUILD_BOOGIE=1
 BUILD_CORRAL=1
-BUILD_SYMBOOGLIX=1
```

Comment:
Don't remove Symbooglix. I think you are not looking at the latest develop maybe.